Console/retro/ConsoleWindow.cc

```diff
@@ -26,13 +26,19 @@
 #include <unordered_map>
 #include <cstring>
 #include <TextUtils.h>
+#include <functional>
+#include <OSUtils.h>
+#include <Traps.h>
 
+using namespace std;
 using namespace retro;
 
 namespace
 {
     std::unordered_map<WindowPtr, ConsoleWindow*>    *windows = NULL;
+    function<bool(EventRecord *)> getEvent;
```

Comment:
bool (*getEvent)(EventRecord*) should be enough and more efficient because we don't need any captured variables or anything.